core/system/inc/page_allocator_config.h

```diff
@@ -23,18 +23,18 @@
 
 /* We can only protect a small number of pages efficiently, so there should be
  * a relatively low limit to the number of pages.
- * By default a maximum of 16 pages are allowed. This can only be overwritten
- * by the porting engineer for the current platform. */
+ * By default a certain maximum number of pages are allowed. This can only be
+ * overwritten by the porting engineer for the current platform. */
 #ifndef UVISOR_PAGE_MAX_COUNT
-#define UVISOR_PAGE_MAX_COUNT (16UL)
+#define UVISOR_PAGE_MAX_COUNT (128UL)
```

Comment:
I believe @meriac wanted this to stay low. This results in  (128 + 8 + 31) / 32) = 5 * `uint32_t` * #boxes (something like 5 * (5 + 1) * 4 = 30 * 4 = 120B).